src/events/user.js

```diff
@@ -0,0 +1,78 @@
+import { EventEmitter } from 'events';
+import debug from 'debug';
+import moment from 'moment';
+
+const dlog = debug('that:api:sessions:events:user');
+
+function userEvents(postmark) {
+  const userEventEmitter = new EventEmitter();
+  dlog('user event emitter created');
+
+  function newSessionCreated({ user, session }) {
+    dlog('new session created');
+
+    postmark.sendEmailWithTemplate({
+      TemplateId: 15581327,
```

Comment:
Comment out TemplateId for now. We are using Alias. Template Id was left there in case there is an issue with alias. The alias will be more flexible for us between environments.